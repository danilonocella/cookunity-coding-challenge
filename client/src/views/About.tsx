import { FC } from "react";
import { Grid, Typography } from "@mui/material";

const Home: FC = () => {
  return (
    <Grid container spacing={2}>
      <Grid item xs={12}>
        <Typography variant="h3" gutterBottom>
          Engineer Coding Challenge
        </Typography>
      </Grid>
      <Grid item xs={12}>
        <Typography variant="h4" gutterBottom>Purpose</Typography>
      </Grid>
      <Grid item xs={12}>
        <Typography>
          As part of CookUnity's selection process, CookUnity requests that candidates
          complete a small project aimed at helping grasp their skills,
          experiences, creativity, problem-solving abilities, and knowledge.
        </Typography>
      </Grid>
    </Grid>
  );
};

export default Home;
