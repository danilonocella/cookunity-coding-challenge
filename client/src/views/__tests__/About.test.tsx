import { render } from "@testing-library/react";
import About from "../About";

test("About component matches snapshot", () => {
  const { container } = render(
    <About />
  );
  expect(container).toMatchSnapshot();
});
