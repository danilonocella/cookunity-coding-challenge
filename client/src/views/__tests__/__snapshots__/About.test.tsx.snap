// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
    >
      <h3
        class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-1mktp8m-MuiTypography-root"
      >
        Engineer Coding Challenge
      </h3>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
    >
      <h4
        class="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom css-1vw6mcs-MuiTypography-root"
      >
        Purpose
      </h4>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
      >
        As part of CookUnity's selection process, CookUnity requests that candidates complete a small project aimed at helping grasp their skills, experiences, creativity, problem-solving abilities, and knowledge.
      </p>
    </div>
  </div>
</div>
`;
